ProcessMaker/Templates/ProcessTemplate.php

```diff
@@ -17,6 +17,7 @@
 use ProcessMaker\Models\ProcessCategory;
 use ProcessMaker\Models\ProcessTemplates;
 use ProcessMaker\Models\Template;
+use ProcessMaker\Package\Projects\Models\ProjectAsset;
```

Comment:
Hey @mcraeteisha 

Rather than importing the class directly here, considering that 'Projects' is an Enterprise package and may not be available on all instances, I suggest creating a constant variable as follows:

`const PROJECT_ASSET_MODEL_CLASS = 'ProcessMaker\Package\Projects\Models\ProjectAsset';`

Then we can wrap lines 266 - 279 in an 'if' statement, checking for the existence of the class:
```
if (class_exists(self::PROJECT_ASSET_MODEL_CLASS)) {
 // lines 266-279 here
}
```
This approach should help us maintain compatibility across different instances.